src/AppBundle/Entity/CommitteeFeedItem.php

```diff
@@ -98,11 +98,41 @@ public static function createEvent(
 
     public function getContent(): ?string
     {
+        if ($this->event instanceof CommitteeEvent) {
+            return $this->event->getDescription();
```

Comment:
Nope because if there is a flush() at some point, the entity will be updated in the DB.w